packages/cli/src/webhooks/webhook-request-handler.ts

```diff
@@ -52,8 +54,13 @@ class WebhookRequestHandler {
 					response.headers,
 				);
 			}
-		} catch (error) {
-			return ResponseHelper.sendErrorResponse(res, error as Error);
+		} catch (e) {
+			const error = ensureError(e);
+			Container.get(Logger).debug(
+				`Error in handling webhook request ${req.method} ${req.path}: ${error.message}`,
```

Comment:
We could add the stack trace as metadata?

Reply:
Good idea 👍 Added